store/tikv/mock-tikv/cop_handler_dag.go

```diff
@@ -281,7 +283,8 @@ func (e *evalContext) decodeRelatedColumnVals(relatedColOffsets []int, handle in
 			}
 			continue
 		}
-		row[offset], err = tablecodec.DecodeColumnValue(value[offset], e.fieldTps[offset])
+		// TODO: Should use session's TimeZone instead of UTC.
```

Comment:
There is a `TimeZoneOffset` field in `tipb.SelectRequest` and `tipb.DAGRequest`, we should use them.